lib/ui/view/profile_page/profile_page_viewmodel.dart

```diff
@@ -25,8 +27,8 @@ class ProfilePageViewModel extends BaseViewModel {
         // the username should be replaced with the username from the rest api
         name: 'User Name',
       );
-      print(result);
-      print(response.statusCode);
+      log.i(result);
+      log.i(response.statusCode);
```

Comment:
I think dio have a log already base on the response, no need to have a log in the viewmodel

Reply:
Alright. I will check it out